src/Pages/sanghunlee/Main/Main.js

```diff
@@ -0,0 +1,314 @@
+import React, { Component } from 'react';
+import Nav from '../../../Components/Nav/Nav';
+import Feed from './Feed';
+import '../styles/Common.scss';
+import './Main.scss';
+
+class Main extends Component {
+  constructor() {
+    super();
+    this.state = {
+      feedList: [],
+      inputValue: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/feedData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          feedList: data,
+        });
+      });
+  }
+
+  getCommentValue = e => {
+    this.setState({
+      inputValue: e.target.value,
+    });
+  };
+
+  postComment = feedId => {
+    if (this.state.inputValue.length > 0) {
+      let newFeedList = Array.from(this.state.feedList);
```

Comment:
- 재할당 되는 변수가 아니라면 const를 사용해서 선언해주세요!
- spread 연산자도 활용해보세요!
```suggestion
      const newFeedList = [...this.state.feedList];
```